tsconfig.base.json

```diff
@@ -23,7 +23,7 @@
     "strictFunctionTypes": true,
     "strictNullChecks": true,
     "strictPropertyInitialization": true,
-    "target": "ESNext",
+    "target": "ES2022",
```

Comment:
what is the status of decorators in ECMAScript/TypeScript? Does the compiled output run in the browser/node 16+?

Reply:
Yes, it does!